### ISSUE TITLE
Meta data

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Khang-NT/aacdecoder-android#19]
```
Audio stream has meta data which may contains title of playing song or other. 
It is possible to read it meta data in decoder and then notify UI when meta 
data was received or changed?
```

Original issue reported on code.google.com by `NKuznet...@gmail.com` on 18 Jul 2012 at 4:43

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
This is already possible using the interface PlayerCallback. See the 
AACPlayerActivity.java example implementation.

   /**
     * This method is called when the player receives a metadata information.
     * It can be either before starting the player (from HTTP header - all header pairs)
     * or during playback (metadata frame info).
     *
     * The list of available keys is not part of this project -
     * it is depending on the server implementation.
     *
     * @param key the metadata key - e.g. from HTTP header: "icy-genre", "icy-url", "content-type",..
     *      or from the dynamic metadata frame: e.g. "StreamTitle" or "StreamUrl"
     * @param value the metadata value
     */
    public void playerMetadata( String key, String value );
```

Original comment by `vbarta...@gmail.com` on 23 Jul 2012 at 10:25

**Comment by GoogleCodeExporter** [comment]
```
Already implemented.
```

Original comment by `vbarta...@gmail.com` on 13 Aug 2012 at 10:26
- Changed state: **Invalid**
